lib/src/key/mod.rs

```diff
@@ -15,7 +15,7 @@
 ///
 /// crate::key::database::all            /*{ns}*{db}
 /// crate::key::database::az             /*{ns}*{db}!az{az}
-/// crate::key::database::cf             /*{ns}*{db}!cf{ts}
+/// crate::key::database::cf             /*{ns}*{db}#{ts}
```

Comment:
Actually I think this line should have been deleted previously. This key no longer exists (it is now `lib/src/key/change/mod.rs`).

Reply:
Thanks @tobiemh! Addressed in 42d99ac4